### PR TITLE
Added Team Section

### PR DESCRIPTION
**Pull request by vedant-jain03**
## Related Issuse

- Info about Issue or bug

Closes: #279 

#### Describe the changes you've made

I have used Github API of this repo and using Use effect I have fetch the data of Contributors and Mentors.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by vedant-jain03** [comment]
@codewithvk I have added Team Section In about us Please review.
I am sure it will satisfy you.

**Comment by codewithvk** [comment]
@vedant-jain03 Can you attach a screenshot, please!

File: client/src/components/Team/team.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            githublink: 'https://github.com/codewithvk',
+            image: 'https://avatars.githubusercontent.com/u/61119120?v=4'
+        }
+    ];
+    const [contributors, setContributors] = useState([]);
+    useEffect(() => {
+        fetch("https://api.github.com/repos/purnima143/Kurakoo/contributors?per_page=1000")
+          .then(res => res.json())
+          .then(data => {
+            console.log(data);
```
Comment (codewithvk) [inline client/src/components/Team/team.jsx:37]: Here mentor name also repeated so remove the mentors and project admin name from the contributor.

File: client/src/components/Team/team.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            setContributors(
+              data.filter(element => !(element.login === "smaranjitghose" || element.login === "anushbhatia"))
```
Comment (codewithvk) [inline client/src/components/Team/team.jsx:39]: I am not getting this line, Can you elaborate more?

**Review by codewithvk — CHANGES_REQUESTED** [review]

**Review by vedant-jain03 — COMMENTED** [review]
Filtered

**Comment by vedant-jain03** [comment]
@codewithvk I have used 
data.filter function so that mentors and owner id should not show again.

**Comment by vedant-jain03** [comment]
https://user-images.githubusercontent.com/76901313/116799750-b6d66200-ab18-11eb-9280-0486231982c8.mp4

Have a look at this. @codewithvk

**Pull request merged**